### DISCUSSION TITLE
AA wireless without dongle

### DISCUSSION THREAD

**Discussion by OB1Jacobi** [agronick/aa-torque#121] [Q&A]
I'm currently using a Pixel 7 on Android 14. So far the only reports I've seen of people consistently making this work is to get an AA wireless dongle and enable developer mode. I have a 2022 Chevrolet Bolt EV with built in wireless Android Auto. Is there anyway I can make this work without having to purchase a dongle I don't need?

**Comment by Vince56190** [comment]
No , only with AAwireless
